app/views/admin/edition_legacy_associations/edit.html.erb

```diff
@@ -29,11 +29,15 @@
         <legend>Associations</legend>
 
         <div class="important-associations">
-          <% if @edition.can_be_related_to_policies? %>
+          <% if @edition.can_be_related_to_policies? && Policy.all.any? %>
             <%= render 'admin/shared/policy_fields', form: form %>
+          <% else %>
+            <p>There are no Policies to associate</p>
```

Comment:
Suggestions on what to write here?

Reply:
I'd just change "associate" to "tag to" (as that makes a bit more sense to me), and then let's talk to Si and Roxanne before this appears to users.